tests/_utils/start_tidb_cluster

```diff
@@ -50,28 +63,45 @@ cat - > "$OUT_DIR/tidb-config.toml" <<EOF
 split-table = true
 EOF
 
-echo "Starting TiKV..."
+echo "Starting Upstream TiKV..."
 tikv-server \
-    --pd ${PD_HOST}:${PD_PORT} \
-    -A ${TIKV_HOST}:${TIKV_PORT} \
+    --pd ${UP_PD_HOST}:${UP_PD_PORT} \
+    -A ${UP_TIKV_HOST}:${UP_TIKV_PORT} \
     --log-file "$OUT_DIR/tikv.log" \
     -C "$OUT_DIR/tikv-config.toml" \
     -s "$OUT_DIR/tikv" &
 
+echo "Starting Downstream TiKV..."
+tikv-server \
+    --pd ${DOWN_PD_HOST}:${DOWN_PD_PORT} \
+    -A ${DOWN_TIKV_HOST}:${DOWN_TIKV_PORT} \
+    --log-file "$OUT_DIR/tikv_down.log" \
+    -C "$OUT_DIR/tikv-config.toml" \
+    -s "$OUT_DIR/tikv_down" &
+
 sleep 2
 
-echo "Starting TiDB..."
+echo "Starting Upstream TiDB..."
 tidb-server \
-    -P ${US_TIDB_PORT} \
+    -P ${UP_TIDB_PORT} \
     -config "$OUT_DIR/tidb-config.toml" \
     --store tikv \
-    --path ${PD_HOST}:${PD_PORT} \
-    --status=${US_TIDB_STATUS} \
+    --path ${UP_PD_HOST}:${UP_PD_PORT} \
+    --status=${UP_TIDB_STATUS} \
     --log-file "$OUT_DIR/tidb.log" &
 
-echo "Verifying TiDB is started..."
+echo "Starting Downstream TiDB..."
+tidb-server \
+    -P ${DOWN_TIDB_PORT} \
+    -config "$OUT_DIR/tidb-config.toml" \
+    --store tikv \
```

Comment:
Is there any specific purpose to start a complete TiDB/TiKV/PD cluster, instead of using a simple mock-tikv storage

Reply:
test recover table ddl,
when we execute recover table ddl on TiDB with mock-tikv, we will get a error.